Method,Name,Error,Occurrences
GET,/category.html,"HTTPConnectionPool(host='39.100.0.61', port=30001): Max retries exceeded with url: /category.html (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fb4b87c66a0>: Failed to establish a new connection: [Errno 111] Connection refused'))",2
GET,/detail.html?id=510a0d7e-8e83-4193-b483-e27e09ddc34d,"HTTPConnectionPool(host='39.100.0.61', port=30001): Max retries exceeded with url: /detail.html?id=510a0d7e-8e83-4193-b483-e27e09ddc34d (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fb4b8659550>: Failed to establish a new connection: [Errno 111] Connection refused'))",1
POST,/orders,"HTTPConnectionPool(host='39.100.0.61', port=30001): Max retries exceeded with url: /orders (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fb4b87449e8>: Failed to establish a new connection: [Errno 111] Connection refused'))",4
POST,/cart,500 Server Error: Internal Server Error for url: http://39.100.0.61:30001/cart,47
GET,/catalogue,"HTTPConnectionPool(host='39.100.0.61', port=30001): Max retries exceeded with url: /catalogue (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fb4b868ad68>: Failed to establish a new connection: [Errno 111] Connection refused'))",1
GET,/detail.html?id=a0a4f044-b040-410d-8ead-4de0446aec7e,"HTTPConnectionPool(host='39.100.0.61', port=30001): Max retries exceeded with url: /detail.html?id=a0a4f044-b040-410d-8ead-4de0446aec7e (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fb4b86590b8>: Failed to establish a new connection: [Errno 111] Connection refused'))",1
GET,/login,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))",2
GET,/detail.html?id=808a2de1-1aaa-4c25-a9b9-6612e8f29a38,"HTTPConnectionPool(host='39.100.0.61', port=30001): Max retries exceeded with url: /detail.html?id=808a2de1-1aaa-4c25-a9b9-6612e8f29a38 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fb4b85b6160>: Failed to establish a new connection: [Errno 111] Connection refused'))",1
GET,/customers,"HTTPConnectionPool(host='39.100.0.61', port=30001): Max retries exceeded with url: /customers (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fb4b8769e10>: Failed to establish a new connection: [Errno 111] Connection refused'))",2
GET,/detail.html?id=3395a43e-2d88-40de-b95f-e00e1502085b,"HTTPConnectionPool(host='39.100.0.61', port=30001): Max retries exceeded with url: /detail.html?id=3395a43e-2d88-40de-b95f-e00e1502085b (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fb4b86a6588>: Failed to establish a new connection: [Errno 111] Connection refused'))",1
POST,/orders,406 Client Error: Not Acceptable for url: http://39.100.0.61:30001/orders,1286
GET,/detail.html?id=819e1fbf-8b7e-4f6d-811f-693534916a8b,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))",1
DELETE,/cart,"HTTPConnectionPool(host='39.100.0.61', port=30001): Max retries exceeded with url: /cart (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fb4b87fa390>: Failed to establish a new connection: [Errno 111] Connection refused'))",3
POST,/cart,"HTTPConnectionPool(host='39.100.0.61', port=30001): Max retries exceeded with url: /cart (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fb4b864df60>: Failed to establish a new connection: [Errno 111] Connection refused'))",3
GET,/basket.html,"HTTPConnectionPool(host='39.100.0.61', port=30001): Max retries exceeded with url: /basket.html (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fb4b8659d30>: Failed to establish a new connection: [Errno 111] Connection refused'))",1
DELETE,/cart,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))",1
